src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/EditInvitation/CreateInvitationCommandHandlerTests.cs

```diff
@@ -19,6 +19,9 @@ public class EditInvitationCommandHandlerTests
         private Mock<IInvitationRepository> _invitationRepositoryMock;
 
         private readonly string _plant = "PCS$TEST_PLANT";
```

Comment:
Wrong name in the solution explorer, should be EditInvitationCommandHandlerTests?